<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>welcomescreen</string>
  </property>
  <widget class="QWidget" name="bg_widget" native="true">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>10</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bg_widget{
background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.29052 rgba(0, 152, 0, 255), stop:0.957187 rgba(0, 0, 0, 255));}</string>
   </property>
   <widget class="QLabel" name="ventalis_title">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>281</width>
      <height>50</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;Century Schoolbook&quot;; color :rgb(0, 0, 0);
font: 36pt &quot;Sylfaen&quot;;</string>
    </property>
    <property name="text">
     <string>VENTALIS</string>
    </property>
   </widget>
   <widget class="QLabel" name="soustitle">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;Century Schoolbook&quot;; color :rgb(0, 0, 0);
font: 22pt &quot;Sylfaen&quot;;</string>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>275</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="email_label">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pwd_label">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Mot de passe</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLineEdit" name="pwd_input">
       <property name="styleSheet">
        <string notr="true">background-color : rgba(0,0,0,0);
font: 14pt &quot;MS Shell Dlg 2&quot;;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-style: solid;
border-width: 2px;
border-color: rgb(0,255, 0);  /* Exemple : bordure verte */</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="email_input">
       <property name="styleSheet">
        <string notr="true">background-color : rgba(0,0,0,0);
font: 14pt &quot;MS Shell Dlg 2&quot;;
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-style: solid;
border-width: 2px;
border-color: rgb(0,255, 0);  /* Exemple : bordure verte */
</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="email_error">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;
color:red;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="pwd_error">
       <property name="styleSheet">
        <string notr="true">color: red;
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="connexion_btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(0, 255, 0);
background-color: black;

font: 14pt &quot;MS Shell Dlg 2&quot;;
border-color: white;
border-style: solid;
border-style: solid;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string>Se connecter</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
